apps/web/src/pages/integrations/components/multi-provider/SelectPrimaryIntegrationModal.tsx

```diff
@@ -152,7 +155,7 @@ export const SelectPrimaryIntegrationModal = ({
       return el.channel === channelType && isNotExcluded;
     });
 
-    return filteredIntegrations.map((el) => mapToTableIntegration(el, environments));
+    return filteredIntegrations.filter((el) => el.active).map((el) => mapToTableIntegration(el, environments));
```

Comment:
Shouldn't we allow to see the not active ones? Because if they select them as primary we also change them to active.

Reply:
This was a requirement in the issue... @LetItRock maybe you have more info here.